src/client/builtin_scene/shaders/mesh.vert

```diff
@@ -42,6 +42,9 @@ in vec4 instanceColor;
 in vec2 instanceTexUvOffset;
 in vec2 instanceTexUvScale;
 in uint instanceLayerIndex;
+in vec2 instanceTexUvOffsetRight;
+in vec2 instanceTexUvScaleRight;
+in uint instanceLayerIndexRight;
```

Comment:
Because the left and right images(actually one image) are uploaded in a single `texSubImage()` call, it's impossible to have a different layer index for right view.